OneSignalSDK/onesignal/src/main/java/com/onesignal/OSInAppMessageController.java

```diff
@@ -750,6 +762,24 @@ private void displayMessage(@NonNull final OSInAppMessage message) {
         }
 
         inAppMessageShowing = true;
+        waitForTags = false;
+        if (message.getHasLiquid()) {
```

Comment:
We definitely could do that and it would be cleaner code but this is theoretically more performant since we are kicking off the rest requests in parallel we don't need to wait for 2 round trips. 

I am open to not doing them in parallel if the performance improvement is less important than the simplicity and reliability of the sequential calls.

Reply:
Agree that this will be less performant, but I'm worried about future extensibility and maintainability with other functionalities. Maybe @jkasten2 can give his opinion on this one

Reply:
Left a follow up comment here: https://reviewable.io/reviews/onesignal/onesignal-android-sdk/1283/OneSignalSDK/onesignal/src/main/java/com/onesignal/OSInAppMessageController.java#-MV-jin1cLek6URzCeI5-r1-766